iron-form.html

```diff
@@ -128,10 +135,12 @@
 
       // Holds all the custom elements registered with this form.
       this._customElements = [];
+      // Holds the initial values of the custom elements registered with this form.
+      this._customElementsInitialValues = [];
```

Comment:
That's going to be a different element, the `<iron-form>`, no*? In that case, that's already a thing: https://github.com/PolymerElements/iron-form/issues/46

* a type extension can't listen to its children changing like this, can it?